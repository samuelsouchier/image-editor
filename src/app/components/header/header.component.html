<header class="header">
  <h1 class="header-title">Simple image editor</h1>
</header>
