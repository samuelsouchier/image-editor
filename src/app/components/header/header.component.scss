@import 'src/styles/colors';
@import 'src/styles/variables';

.header {
  width: 100%;
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: $spacing $spacing-xxl;
  background: $color-primary;
  color: $color-background;
}
