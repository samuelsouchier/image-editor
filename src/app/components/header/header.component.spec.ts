import { createComponentFactory, Spectator } from '@ngneat/spectator';

import { HeaderComponent } from './header.component';

describe(HeaderComponent.name, () => {
  let spectator: Spectator<HeaderComponent>;
  const createComponent = createComponentFactory({
    component: HeaderComponent,
    shallow: true,
  });

  it('should match snapshot', () => {
    spectator = createComponent();

    expect(spectator.fixture).toMatchSnapshot();
  });
});
