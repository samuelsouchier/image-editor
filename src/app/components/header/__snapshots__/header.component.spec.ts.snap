// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent should match snapshot 1`] = `
<app-header>
  <header
    class="header"
  >
    <h1
      class="header-title"
    >
      Simple image editor
    </h1>
  </header>
</app-header>
`;
